src/loaders/anndata-loaders/MatrixZarrLoader.js

```diff
@@ -79,7 +83,7 @@ export default class MatrixZarrLoader extends AbstractTwoStepLoader {
    * @returns {Number} The number of cells.
    */
   async _getNumCells() {
-    const cells = await this.dataSource.loadCellNames();
+    const cells = await this.dataSource.loadObsIndex();
```

Comment:
Jumping off my comment from above, now that there are two ways of doing the same thing, we have mixed use in the class (`this.dataSource.loadObsIndex()` vs `this.loadCellNames()`)